tests/system/node/0/config.yaml

```diff
@@ -54,4 +54,4 @@ network:
 ################################################################################
 logging:
   # Values: Trace, Info, Warn, Error, Fatal, None (default)
-  level: Trace
+  level: Info
```

Comment:
But why the change from trace to info in these tests? Is there too much vibe.d spam?

Reply:
>  Is there too much vibe.d spam?

It prints like 8 lines for EVERY request...

Reply:
Woops!